doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -239,35 +323,59 @@ NCSDK-20092: ESB does not send packet longer than 63 bytes
 Matter
 ======
 
+There are following new known issues to be added for the Matter:
```

Comment:
I think this line should not be added, as it is just part of my github comment :)

Reply:
It was a copy past issue :). Updated.